content/sensu-go/6.3/observability-pipeline/observe-schedule/rule-templates.md

```diff
@@ -0,0 +1,471 @@
+---
+title: "Rule templates reference"
+linkTitle: "Rule Templates Reference"
+reference_title: "Rule templates"
+type: "reference"
+description: "Sensu's rule templates are resources that Sensu applies to business service components. The rules define the conditions under which Sensu will consider a service component online, degraded, or offline. Read this reference to learn more about Sensu's rule templates for business service management."
+weight: 70
+version: "6.3"
+product: "Sensu Go"
+menu: 
+  sensu-go-6.3:
+    parent: observe-schedule
+---
+
+**COMMERCIAL FEATURE**: Access business service management, including rule templates, in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][4].
+
+Rule templates are resources that Sensu applies to business service [components][3].
+The rules define the conditions under which Sensu will consider a service component online, degraded, or offline.
+Rules apply to aggregate data derived from a service component’s selection of Sensu Go events.
+
+In rule templates, you configure the conditions for creating an event with an ECMAScript 5 (JavaScript) expression.
+When you add rule templates by reference in your service component definitions, Sensu evaluate events according to the monitoring rules you configure in the rule template.
```

Comment:
```suggestion
When you add rule templates by reference in your service component definitions, Sensu evaluate events according to the monitoring rules configured in the rule template.
```

Reply:
I remember Sean saying many users will not need to configure their own rule templates because we are providing some out-of-the-box templates that will meet all of their BSM needs. If I understand that correctly, I want to explain that here -- and should add a section that describes the built-in rule templates.